specification/metrics/semantic_conventions/system-metrics.md

```diff
@@ -33,12 +33,14 @@ instruments not explicitly defined in the specification.
-|                        |                                                                                                          |       |                                                   |            | cpu              | CPU number [0..n-1]                 |
-| system.cpu.utilization | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
-|                        |                                                                                                          |       |                                                   |            | cpu              | CPU number (0..n)                   |
+| Name                      | Description                                                                                              | Units | Instrument Type ([*](README.md#instrument-types)) | Value Type | Attribute Key(s) | Attribute Values                    |
+| ------------------------- | -------------------------------------------------------------------------------------------------------- | ----- | ------------------------------------------------- | ---------- | ---------------- | ----------------------------------- |
+| system.cpu.time           |                                                                                                          | s     | Counter                                           | Double     | state            | idle, user, system, interrupt, etc. |
+|                           |                                                                                                          |       |                                                   |            | cpu              | CPU number [0..n-1]                 |
+| system.cpu.utilization    | Difference in system.cpu.time since the last measurement, divided by the elapsed time and number of CPUs | 1     | Gauge                                             | Double     | state            | idle, user, system, interrupt, etc. |
+|                           |                                                                                                          |       |                                                   |            | cpu              | CPU number (0..n)                   |
+| system.cpu.physical.count | Reports the number of available physical CPUs - may change over time                                | 1     | UpDownCounters                                    | Int64      |                  |                                     |
+| system.cpu.virtual.count  | Reports the number of available virtual CPUs - may change over time                                 | 1     | UpDownCounters                                    | Int64      |                  |                                     |
```

Comment:
Should the unit be `{cpu}`? AFAIR `1` is recommended for ratio metrics

Reply:
changed it, thanks!